### DISCUSSION TITLE
Want to use HPA's container resource metrics

### DISCUSSION THREAD

**Discussion by abhishtfw** [kedacore/keda#2885] [Q&A / Need Help]
I want to pass the container resource metrics config to the HPA created by Keda [https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#container-resource-metrics](url). With the latest version, seems it is not possible. Is there any alternate way to achieve this or any scaler that can help?

**Comment by zroubalik** [comment]
It is not possible atm, we will need to extend existing CPU/Memory scalers. So you can open an issue for tracking this effort and even better if you are willing to contribute this :)

**Comment by aaron-sysdig** [comment]
hey @zroubalik, is this still not supported?
